Sources/core/Game.h

```diff
@@ -103,4 +105,5 @@ public slots:
 
     id_ptr_on<UnsyncGenerator> unsync_generator_;
     id_ptr_on<IMob> current_mob_;
+
 };
```

Comment:
New line should be removed. Overall, these changes are needed because it is harder to read changes when a lot of unnecessary changes present.